Apps/WebClient/src/ClientApp/src/views/errors/idirLoggedIn.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+    <div>
+        <ErrorComponent :error="errorDescription" />
+    </div>
+</template>
+
+<script lang="ts">
+import Vue from "vue";
+import { Component, Prop } from "vue-property-decorator";
+import PageErrorComponent from "@/components/error.vue";
+import PageError from "@/models/pageError";
+
+@Component({
+    components: {
+        ErrorComponent: PageErrorComponent,
+    },
+})
+export default class IdirLoggedInView extends Vue {
+    public errorDescription: PageError = new PageError(
+        "403",
+        "Not BC Services Card login",
```

Comment:
We don't want this text "Not BC Services Card login" as we currently support Keycloak in lower environments and we may add other IDPs in the future.  We just want the text related to IDIR.

Reply:
I think we talked about this last week since it was the other way around. IMO we want to allow only BCSC logins. If we need to allow different ipds in the future we can change this.

Reply:
sounds good, I will leave the title as is for now then